content/en/docs/concepts/storage/volumes.md

```diff
@@ -921,20 +921,15 @@ For more information, see the [vSphere volume](https://github.com/kubernetes/exa
 
 #### vSphere CSI migration {#vsphere-csi-migration}
 
-{{< feature-state for_k8s_version="v1.19" state="beta" >}}
-
-The `CSIMigrationvSphere` feature for `vsphereVolume` is enabled by default as of Kubernetes v1.25.
-All plugin operations from the in-tree `vspherevolume` will be redirected to the `csi.vsphere.vmware.com` {{< glossary_tooltip text="CSI" term_id="csi" >}} driver unless `CSIMigrationvSphere` feature gate is disabled.
+{{< feature-state for_k8s_version="v1.26" state="stable" >}}
 
+All plugin operations from the in-tree `vspherevolume` will be redirected to the `csi.vsphere.vmware.com` {{< glossary_tooltip text="CSI" term_id="csi" >}} driver.
```

Comment:
```suggestion
In Kubernetes {{< skew currentVersion >}}, all operations for the in-tree `vsphereVolume` type
are redirected to the `csi.vsphere.vmware.com` {{< glossary_tooltip text="CSI" term_id="csi" >}} driver.
```
(the feature state shortcode is a strong hint about when this behavior came in).

What happens if the CSI driver isn't working / isn't installed? It'd be nice to explain that bit.

Reply:
added this line 

> If vSphere CSI Driver is not installed volume operations can not be performed on the PV created with the in-tree `vsphereVolume` type.